File: free-platform-book/free-platform-book-starter/src/main/java/top/yangwulang/platform/repository/book/ChapterContentRepository.java
package top.yangwulang.platform.repository.book;

import org.babyfish.jimmer.spring.repository.JRepository;
import org.springframework.stereotype.Repository;
import top.yangwulang.platform.entity.book.ChapterContent;

@Repository
public interface ChapterContentRepository extends JRepository<ChapterContent, String> {
}

File: platform/src/main/kotlin/top/yangwulang/platform/repository/book/ChapterContentRepository.kt
package top.yangwulang.platform.repository.book;

import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.data.jpa.repository.JpaSpecificationExecutor
import top.yangwulang.platform.entity.book.ChapterContent

interface ChapterContentRepository : JpaRepository<ChapterContent, String>, JpaSpecificationExecutor<ChapterContent> {
    fun findByChapterId(chapterId: String): ChapterContent?


    fun existsByChapterId(chapterId: String): Boolean
}